enterprise/suppress-user/setup.go

```diff
@@ -40,11 +42,28 @@ func (m *Factory) Setup(backendConfig backendconfig.BackendConfig) (types.UserSu
 	loadConfig()
 	ctx := context.TODO()
 	backendConfig.WaitForConfig(ctx)
-	workspaceId := backendConfig.Identity().ID()
+	ID := backendConfig.Identity().ID()
```

Comment:
We may use the `identity.Identifier` for everything and set is as a field in `SuppressRegulationHandler`

1. Introduce a new method in `identity.Identifier`: `Type() deployment.Type`
2. `getSourceRegulationsFromRegulationService` can construct the url from `identity.Identifier`
3. `getSourceRegulationsFromRegulationService` can delegate basic authentication to `identity.Identifier`